docker/openproblems-github-actions/r_requirements.txt

```diff
@@ -1,6 +1,12 @@
 backports@1.4.1
+bioc::ComplexHeatmap@2.12.0
+bioc::OmnipathR@3.4.0
+bioc::scater@1.24.0
+bioc::scran@1.24.0
+bioc::scuttle@1.6.2
 docopt@0.7.1
 git2r@0.30.1
 lintr@3.0.0
+saezlab/liana@0.1.5
```

Comment:
@dbdimitrov dependencies for liana are pretty hefty and I don't really want to add a tonne of stuff to this image. How much work would it be to split out the resource generation dependencies from the rest of liana? (as in, have a separate package called `liana-resource`)?